src/NodaTime/Instant.cs

```diff
@@ -655,6 +655,28 @@ public static Instant FromUnixTimeTicks(long ticks)
         public long ToUnixTimeSeconds() =>
             duration.FloorDays * (long) SecondsPerDay + duration.NanosecondOfFloorDay / NanosecondsPerSecond;
 
+        /// <summary>
+        /// Gets the number of seconds since the Unix epoch, along with remaining nanoseconds.
+        /// Negative values represent instants before the Unix epoch.
+        /// </summary>
+        /// <remarks>
+        /// If the number of nanoseconds in this instant is not an exact number of seconds
+        /// the value is truncated towards the start of time, 
```

Comment:
trailing comma? But I think this is confusing anyway. Better might be to say something like "The returned seconds value is always the same value as returned by ToUnixTimeSeconds(). The returned nanoseconds value is always non-negative and in the range [0, 10^9); that is, for an instant 1250ms before the Unix epoch, this function will return (-2 seconds, +750,000,000 nanoseconds)."

... or with better words, but you get the idea: I think it is obvious that this is returning values such that (_seconds_ * 10^9) + _ns_ = _total ns_, but I don't think it's obvious that the representation isn't same-sign (e.g. (-1, -250,000,000) for the above).